dolphin/alert_manager/trap_receiver.py

```diff
@@ -63,6 +64,27 @@ def _add_transport(self):
         except Exception:
             raise ValueError("Port binding failed the provided port is in use.")
 
+    def _extract_oid_value(self, var_bind):
+        """Extracts oid and value from var binds.
+        ex: varbind =  (SNMPv2-MIB::snmpTrapOID.0 = SNMPv2-MIB::authenticationFailure)
+        oid = snmpTrapOID
+        val = authenticationFailure
+        """
+        var_bind_info = var_bind.prettyPrint()
+        var_bind_info = var_bind_info.split("=", 1)
+        oid = var_bind_info[0]
+        val = var_bind_info[1]
+
+        oid = oid.split("::", 1)
+        oid = oid[1].split(".", 1)
+        oid = oid[0]
+
+        if "::" in val:
+            val = val.split("::", 1)
+            val = val[1]
+
```

Comment:
it's better add more comments to explain your logic here.

Reply:
Updated